build.py

```diff
@@ -134,17 +134,17 @@ def get_git_version():
     # this may fail with exceptions
     cmd = 'git', 'describe', '--tags', '--long', '--dirty',
     version = subprocess.check_output(cmd, stderr=subprocess.STDOUT).strip()
```

Comment:
You probably would rather decode early using this: then the `b` prefix is not needed afterwards
```suggestion
    version = subprocess.check_output(cmd, encoding='utf-8', stderr=subprocess.STDOUT).strip()
```

Reply:
Hi, I tried that, but it gives error unexpected keyword arg 'encoding' on pyhon2
It works fine with python3 latest versions, but not with python 2.
Also, this guy faced a same issue: https://github.com/eliben/pycparser/issues/296#issuecomment-444810928

I've done decoding on the next line though, thereby removing need for b prefix

Reply:
Fair enough... we want to drop Python2 support alright!